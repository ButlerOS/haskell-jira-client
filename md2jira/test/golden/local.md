---
assigned: user@corp
---

This is my backlog.

# Develop app

## Implement X

The goal is...
