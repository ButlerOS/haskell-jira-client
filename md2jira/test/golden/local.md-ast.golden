Document
    { config = DocumentConfig { users = Nothing }
    , intro =
        [ Para
            [ Str "This"
            , Space
            , Str "is"
            , Space
            , Str "my"
            , Space
            , Str "backlog."
            ]
        ]
    , epics =
        [ Epic
            { mJira = Nothing
            , title = "Develop app"
            , description = []
            , stories =
                [ Story
                    { mJira = Nothing
                    , title = "Implement X"
                    , status = Nothing
                    , description =
                        [ Para
                            [ Str "The"
                            , Space
                            , Str "goal"
                            , Space
                            , Str "is..."
                            ]
                        ]
                    , assignee = Nothing
                    , points = Nothing
                    , sprint = Nothing
                    , updated = Nothing
                    }
                ]
            }
        ]
    }